store/postgres/src/store.rs

```diff
@@ -297,16 +297,18 @@ impl Store {
                     format!("{}", e),
                 )
             })? {
-            Some(json) => serde_json::from_value::<Entity>(json)
-                .map(Some)
-                .map_err(|e| {
+            Some((json, entity_type)) => {
```

Comment:
I don't understand the logic here: we queried for `entity = :op_entity`, and so `entity_type == op_entity`, or am I missing something? Just sticking with `op_entity` would make that clearer.

Reply:
Thanks! I copy pasted this logic without thinking much.